PlayLoop/PlayerInteractionHandler.js

```diff
@@ -0,0 +1,201 @@
+class PlayerInteractionHandler {
```

Comment:
after a second read-through i have to say, i really like how you implemented this class  👏

Reply:
Thanks!